components/Banner.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components'
+
+const BannerContainer = styled.div`
+  background: linear-gradient(92.58deg, #0defe1 0%, #78ff96 100%);
+  position: absolute;
```

Comment:
I'm wondering if it's possible to shift the hero down such that the banner isn't covering it? or do you think it looks good enough already haha

- removing position: absolute; looks to fix it on mobile but I haven't thoroughly checked this

![image](https://github.com/nwplus/www/assets/62491197/738d2d73-a360-4a1a-87c8-a12bc7a852d9)